test/scripts/start-services.sh

```diff
@@ -2,21 +2,47 @@
 set -eu
 
 # config directory
-configdir=$(mktemp -d -t artemis-config-XXX)
+rm -rf tmp
+mkdir tmp
+configdir=$(pwd)"/tmp"
```

Comment:
I'd rather not keep temporary state in our checkout. It clutters up things. Can we stick to to a static folder under `/tmp` like `/tmp/artemis-config`? That's the place where temporary files go.

Reply:
sure, coming in next commit